config/build/webpack.config.common.js

```diff
@@ -34,7 +35,7 @@ export const createResolveConfigLoader = () => ({
 
 export function config(options) {
     return {
-        devtool: options.dev && 'source-map',
+        devtool: options.dev && 'eval-cheap-module-source-map',
```

Comment:
I tested them all for pobpack-browser and the only one working with chrome and firefox and react-redbox was source-map